x/concentrated-liquidity/swaps.go

```diff
@@ -79,13 +91,14 @@ var (
 // As a result, the range from the end of position one to the beginning of position
 // two has no liquidity and can be skipped.
 func (ss *SwapState) updateSpreadRewardGrowthGlobal(spreadRewardChargeTotal sdk.Dec) {
-	if !ss.liquidity.IsZero() {
-		// We round down here since we want to avoid overdistributing (the "spread factor charge" refers to
-		// the total spread factors that will be accrued to the spread factor accumulator)
-		spreadFactorssAccruedPerUnitOfLiquidity := spreadRewardChargeTotal.QuoTruncate(ss.liquidity)
-		ss.spreadRewardGrowthGlobal.AddMut(spreadFactorssAccruedPerUnitOfLiquidity)
+	if ss.liquidity.IsZero() {
```

Comment:
It's not a very string preference, but I think in this case, but I think it's good to avoid nesting once there are >2 nested blocks (less context to keep in your mental stack when understanding the code)